<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugRegistersWindow</class>
 <widget class="QWidget" name="DebugRegistersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registres</string>
  </property>
  <widget class="QGroupBox" name="groupBoxRegisters8bits">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Registres 8 bits</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelRegisterA">
       <property name="text">
        <string>A = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegisterB">
       <property name="text">
        <string>B = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegisterC">
       <property name="text">
        <string>C = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegisterD">
       <property name="text">
        <string>D = 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelRegisterE">
       <property name="text">
        <string>E = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>F = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegisterH">
       <property name="text">
        <string>H = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>L = 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>40</y>
      <width>31</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxRegisters16bits">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Registres 16 bits</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="labelRegisterAF">
       <property name="text">
        <string>AF = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegisterBC">
       <property name="text">
        <string>BC = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegisterPC">
       <property name="text">
        <string>PC = 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="labelRegisterDE">
       <property name="text">
        <string>DE = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegisterHL">
       <property name="text">
        <string>HL = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegisterSP">
       <property name="text">
        <string>SP = 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxFlags">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Flags</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="labelFlagZ">
       <property name="text">
        <string>Z(1) = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFlagH">
       <property name="text">
        <string>H(3) = 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="labelFlagN">
       <property name="text">
        <string>N(2) = 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFlagC">
       <property name="text">
        <string>C(4) = 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
